File: demo-api/src/main/java/io/fries/demo/api/DemoApi.java
package io.fries.demo.api;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class DemoApi {

    public static void main(final String[] args) {
        SpringApplication.run(DemoApi.class, args);
    }
}

File: demo-api/src/main/kotlin/io/fries/demo/api/DemoApi.kt
package io.fries.demo.api

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan

@SpringBootApplication
@ConfigurationPropertiesScan
class DemoApi

fun main(args: Array<String>) {
    SpringApplication.run(DemoApi::class.java, *args)
}